packages/flutter_availability/pubspec.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
   flutter:
     sdk: flutter
   intl: any
-  rxdart: ^0.28.0
+  rxdart: ^0.27.0
```

Comment:
Why are you downgrading this?

Reply:
Because beep has a dependency problem with version 0.28.0, It requires the updating of several packages in beep and i am unsure if it is possible/allowed. Right now we are not using any feature from 0.28.0 so i decided to lower the minimal version of rxdart

Reply:
Ugh, we should really just update stuff in Beep then. I don't like changing deps in independent packages for one specific app...

But fine for now I guess, this is by far the quickest solution.